src/main/java/rx/Single.java

```diff
@@ -1569,14 +1569,12 @@ public final void onNext(T args) {
      * @param subscriber
      *            the Subscriber that will handle the emission or notification from the Single
      */
-    public final void unsafeSubscribe(Subscriber<? super T> subscriber) {
+    public final Subscription unsafeSubscribe(Subscriber<? super T> subscriber) {
```

Comment:
Unsafesubscribe is called by operators and they don't use the returned Subscription. These unsubscribe the Subscriber.